### PR TITLE
Add missing arguments for memcache procedural style functions

### PR DESCRIPTION
**Pull request by tmotyl**
See source code https://github.com/php/pecl-caching-memcache/blob/master/memcache.c 
and documentation examples on php.net. E.g. http://php.net/memcache_decrement

### CODE REVIEW

File: memcache/memcache.php [hunk 1]
```diff
@@ -435,19 +435,46 @@ function memcache_delete () {}
  */
 function memcache_debug ($on_off) {}
 
-function memcache_get_stats () {}
+/**
+ * @var object $memcache
+ */
+function memcache_get_stats ($memcache) {}
```
Comment (andrey-sokolov) [inline memcache/memcache.php]: looks like this method accepts more than one parameter http://php.net/manual/en/memcache.getstats.php

File: memcache/memcache.php [hunk 2]
```diff
@@ -435,19 +435,46 @@ function memcache_delete () {}
...
 
-function memcache_get_extended_stats () {}
+/**
+ * @var object $memcache
+ */
+function memcache_get_extended_stats ($memcache) {}
```
Comment (andrey-sokolov) [inline memcache/memcache.php]: http://php.net/manual/en/memcache.getextendedstats.php

**Comment by andrey-sokolov** [comment]
Thank you for finding issues and investigating it. Looks like some signatures can be improved. Please double-check and fix it.

**Comment by tmotyl** [comment]
This is not nice. 
Please improve the patch as you wish and merge it. I dont have time to work on this right now.
The problem is that currently when you follow the advices from phpstorm inspection you will end up with broken code.

**Pull request closed without merging**